src/hooks/useKanban.js

```diff
@@ -384,6 +384,24 @@ visitorId = 'visitor_' + Date.now() + '_' + Math.random().toString(36).substr(2,
     }
   };
 
+  // Adicionar nova função para atualizar o tipo de atividade
+  const updateActivityType = (columnId, cardId, activityType) => {
+    setColumns(prev => {
```

Comment:
![kody code-review](https://img.shields.io/badge/kody-code--review-brightgreen)

```javascript
const updateActivityType = (columnId, cardId, activityType) => {
  if (!columnId || !cardId) return;
  setColumns(prev => {
    const column = prev[columnId];
    if (!column) {
      console.error('Column not found:', columnId);
      return prev;
    }
```
The function lacks error handling and column existence validation, which could lead to runtime errors if the columnId is invalid.



<sub>Talk to Kody by mentioning @kody</sub>

<sub>Was this suggestion helpful? React with 👍 or 👎 to help Kody learn from this interaction.</sub><!-- kody-codereview -->&#8203;
&#8203;